x-pack/plugins/actions/server/actions_client.ts

```diff
@@ -11,9 +11,16 @@ import {
   SavedObject,
 } from 'src/core/server';
 
+import { i18n } from '@kbn/i18n';
 import { ActionTypeRegistry } from './action_type_registry';
 import { validateConfig, validateSecrets } from './lib';
-import { ActionResult, FindActionResult, RawAction } from './types';
+import { ActionResult, FindActionResult, RawAction, PreConfiguredAction } from './types';
+import { PreconfiguredActionDisabledModificationError } from './lib/errors/preconfigured_action_disabled_modification';
+
+// We are assuming there won't be many actions. This is why we will load
+// all the actions in advance and assume the total count to not go over 100 or so.
+// We'll set this max setting assuming it's never reached.
+export const MAX_ACTIONS_RETURNED = 10000;
```

Comment:
The comment is no longer aligned with the value :) (100 != 10000)

To avoid this in the future it might be worth adding a test instead of a comment - test can't get out of date without being explicitly updated. :)

Reply:
I think it might be worth just removing the number from the comment and replace it with `to not go over this limit`. I still find it valuable having a comment explaining why this limit is in place.